source/llassetgen/source/Packing.cpp

```diff
@@ -0,0 +1,88 @@
+#include <llassetgen/Packing.h>
+
+#include <algorithm>
+#include <tuple>
+
+namespace llassetgen {
+    Packing::Packing(Vec2<PackingSizeType> _atlasSize) : atlasSize{_atlasSize} {}
+
+    namespace impl_packing {
```

Comment:
Copying my comment from your older comment (which Github unfortunately hides because I changed something 2 lines above):

The namespace was originally named `impl`, but I changed it to prevent potential name clashes should we use such a namespace in other headers and didn't really thought much about the name. I now think that that is probably just overcautious, and that we should just stick with `impl` or `detail`. The most important part is that the name makes clear that the contents of the namespace are implementation details and not part of the libraries interface.

`impl` and `detail` is also just the naming scheme I know from other C++ libraries, not sure if there is any best/most common choice here.

Reply:
ah, thanks!

Reply:
Changed to `namespace internal`.